app/views/crime_applications/_current_assignment.html.erb

```diff
@@ -12,28 +12,37 @@
       <strong><%= t(:date_received, scope: 'crime_applications.labels') %></strong>
       <%= crime_application.submitted_at.strftime('%d %B %Y') %>
     </div>
+    <% if crime_application.status == 'returned' %>
+      <div class="govuk-row">
+        <strong><%= t(:date_returned, scope: 'crime_applications.labels') %></strong>
+        <%# TODO implement returned at %>
+        24 October 2022
```

Comment:
Now available in the json of a returned application: application.return_details.returned_at